llvm/include/llvm-c/Core.h

```diff
@@ -386,6 +386,12 @@ typedef enum {
   LLVMAtomicRMWBinOpFMin, /**< Sets the value if it's smaller than the
                            original using an floating point comparison and
                            return the old one */
+  LLVMAtomicRMWBinOpFMaximum, /**< Sets the value if it's greater than the
+                           original using an floating point comparison and
+                           return the old one */
+  LLVMAtomicRMWBinOpFMinimum, /**< Sets the value if it's smaller than the
+                           original using an floating point comparison and
+                           return the old one */
```

Comment:
This needs to be added to the end of the enum to prevent a C ABI break.

Reply:
Thanks, now moved in latest patch.